docs/FAQ.md

```diff
@@ -39,8 +36,7 @@ release.
 
 ### What hardware is supported?
 
-The plugin is tested and supported on P100, V100, T4, A2, A10, A30, A100 and L4 datacenter GPUs.  It is possible
-to run the plugin on GeForce desktop hardware with Volta or better architectures.  GeForce hardware
+Please see [Hardware Requirements](download.md#hardware-requirements) section for the list of GPUs that the RAPIDS plugin has been tested on. It is possible to run the plugin on GeForce desktop hardware with Volta or better architectures.  GeForce hardware
```

Comment:
For each line, we normally limit it to only 100 characters. Could you split it into multiple lines?